index.html

```diff
@@ -0,0 +1,462 @@
+<!--HOME PAGE START-->
+<!doctype html>
+<html lang="en">
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <title>HOME - Techno India NJR Institute Of Technology</title>
+</head>
+<body>
+
+  <!--Carousel-->
+  <div id="image_bar" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#image_bar" data-slide-to="0" class="active"></li>
+      <li data-target="#image_bar" data-slide-to="1"></li>
+      <li data-target="#image_bar" data-slide-to="2"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/Nao-copy-copy.jpg" alt="NAO in Techno India NJR Udaipur">
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/som-copy1.jpg" alt="Somnath Sharma in Microsoft">
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/akshay-copy1.jpg" alt="Akshay Nandwana at Google IO">
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="http://technonjr.org/wp-content/uploads/2015/03/slider00011.jpg " alt="Techno India NJR Udaipur Campus">
+      </div>
+    </div>
+  </div>
+
+  <!--achievements-->
+<section id="achievements" style="padding-top:10%;">
```

Comment:
```suggestion
<section id="achievements">
```
use external css instead